doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -54,18 +53,13 @@ The procedure to install CORTX on OVA is mentioned below.
 
 #. **For Oracle VM VirtualBox Users ONLY**:
```

Comment:
you can add drop down option for this section

Reply:
Added the drop-down option for Oracle VB.